### ISSUE TITLE
'Null' is not a subtype of type 'String'

### ISSUE THREAD

**Issue by ZahaanMahajan** [ValentinVignal/themed_color_palette#10]
After running the command: `flutter pub run build_runner build --delete-conflicting-outputs`

The following error occurred:

```
Deprecated. Use `dart run` instead.
Building package executable... (2.3s)
Built build_runner:build_runner.
[INFO] Generating build script completed, took 176ms
[INFO] Precompiling build script... completed, took 2.3s
[INFO] Building new asset graph completed, took 650ms
[INFO] Checking for unexpected pre-existing outputs. completed, took 0ms
[SEVERE] themed_color_palette:themedColorPaletteGeneratorBuilder on lib/theme.theme.yaml:

type 'Null' is not a subtype of type 'String' in type cast
package:themed_color_palette/src/themed_color_palette.dart 38:36  new ColorPalette._fromJson
package:themed_color_palette/src/themed_color_palette.dart 33:25  new ColorPalette.fromJson
package:themed_color_palette/themed_color_palette.dart 33:39      ThemedColorPaletteBuilder.build

[INFO] Running build completed, took 30ms
[INFO] Caching finalized dependency graph completed, took 36ms
[SEVERE] Failed after 71ms
```

**Comment by ValentinVignal** [comment]
It looks like you don't have a key `.version` at the root of the file.

I should consider making it optional. In the meantime, you should be able to workaround this error by adding

```yaml
.version: 1
```

at the root of the file

**Comment by ValentinVignal** [comment]
It should be fixed in the version  0.4.2
